core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Path2afp.java

```diff
@@ -2062,7 +2067,8 @@ static boolean computeControlPointBoundingBox(PathIterator2afp<?> iterator,
 			default:
 			}
 		}
-		if (foundOneControlPoint) {
			box.setFromCorners(xmin, ymin, xmax, ymax);
+		if (foundOneControlPoint) {
```

Comment:
The `+` character is a problem of formatting for the Javadoc.

Actually, the checkstyle module is configured for assuming 4 spaces per tab character. In the project, we are using the tab characters and not spaces. Indeed, we are considered this approach more convenient for edition. We are differing from the Sonar rules on this point. You should fix your IDE configuration for using tab characters.